scripts/propka3.py

```diff
@@ -3,12 +3,12 @@
 
 This is the original propka script. However, this distribute-based
```

Comment:
Is this script really needed? It could be deleted, couldn't it?

Reply:
It could be but I'm worried about breaking others' workflows in serious ways.  Some people really don't like `python -m <module>`...

Reply:
But this script is never installed. When you run
```
python setup.py install
```
or
```
pip install propka
```
then the wrapper script `propka3` is installed that uses `propka.main.run https://github.com/jensengroup/propka-3.1/blob/0cb470fbdbc9e57c4487e421b9110975d21799ce/setup.py#L53-L57

By having this script here, we just provide uncertainty to users what to use and encourage them _not_ to install via the recommended methods. Furthermore, you have to maintain the same code in multiple locations.

Reply:
I agree with removing the script. The entry point should be a sufficient replacement.

Reply:
Addressed in 2f47d0ea9a5884b633d1c0ffcc7ec5cb6ecbdfc6